c2corg_ui/static/js/documentediting.js

```diff
@@ -354,5 +382,204 @@ app.DocumentEditingController.prototype.getCoordinatesFromPoint_ = function(
 };
 
 
+/**
+ * Clone input
+ * @export
+ */
+
+app.DocumentEditingController.prototype.addAnotherMapsInfo = function() {
+  var template = $('#maps_info-group input:last-of-type').val('').clone();
+  template.insertBefore($('#document-add-maps'));
+}
+
+
+/**
+ * Navigate through creation/editing steps
+ * @param {number} step
+ * @param {string} direction
+ * @export
+ */
+
+app.DocumentEditingController.prototype.step = function(step, direction) {
```

Comment:
it works for 5 or less steps. You're right saying that it should be the most generic and work for event 100 steps, but I've created max 5 divs with 18% of width of the parent div that only slides to the left or right. My argument is that 5 steps is already a lot and we should not have more than that :)